src/core/libraries/kernel/file_system.cpp

```diff
@@ -310,6 +310,37 @@ int PS4_SYSV_ABI posix_mkdir(const char* path, u16 mode) {
     return result;
 }
 
+int PS4_SYSV_ABI sceKernelRmdir(const char* path, u16 mode) {
+    auto* mnt = Common::Singleton<Core::FileSys::MntPoints>::Instance();
+    bool ro = false;
+
+    const std::filesystem::path dir_name = mnt->GetHostPath(path, &ro);
+
+    if (dir_name.empty()) {
+        return SCE_KERNEL_ERROR_EACCES;
+    }
+
+    if (ro) {
+        return SCE_KERNEL_ERROR_EROFS;
+    }
+
+    if (!std::filesystem::is_directory(dir_name)) {
+        return ORBIS_KERNEL_ERROR_ENOTDIR;
+    }
+
+    if (!std::filesystem::exists(dir_name)) {
+        return ORBIS_OK;
```

Comment:
that should return ORBIS_KERNEL_ERROR_ENOENT

Reply:
Sorry, I interpreted this as a case where removing the directory was unnecessary as it already doesn't exist